src/XMakeBuildEngine/UnitTests/BuildEnvironmentHelper_Tests.cs

```diff
@@ -105,7 +105,7 @@ public void BuildEnvironmentDetectsVisualStudioByEnvironment()
             {
                 Environment.SetEnvironmentVariable("VSINSTALLDIR", env.TempFolderRoot);
                 Environment.SetEnvironmentVariable("VisualStudioVersion", "15.0");
-                BuildEnvironmentHelper.ResetInstance_ForUnitTestsOnly();
+                BuildEnvironmentHelper.ResetInstance_ForUnitTestsOnly(getAppContextBaseDirectory: ReturnNull);
```

Comment:
Do you need to specify this? Seems like it should be redundant (since it's after the env-var fallback).